thz/PortHandler.py

```diff
@@ -0,0 +1,274 @@
+#########################################################################
+#  Ported by toggle, 2014
+#########################################################################
+#  This software is based on the FHEM implementation
+#  https://github.com/mhop/fhem-mirror/blob/master/fhem/FHEM/00_THZ.pm
+#
+#  THZ plugin for SmartHome.py.   http://mknx.github.com/smarthome/
+#
+#  This plugin is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This plugin is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this plugin. If not, see <http://www.gnu.org/licenses/>.
+#########################################################################
+
+# This file implements the low level protocol of the THZ serial port.
+#
+
+import serial
+import threading
+import select
+import multiprocessing
+import time
+import sys
+import logging
+
+logger = logging.getLogger('THZ')
```

Comment:
Please move the logger into the class `PortHandler`.

Reply:
Inside the class the logger should be initialized this way:
```
        self.logger = logging.getLogger(__name__)
```

This way the logger is named `plugins.thz`. This standardization is helpful for an upcoming graphical logging configuration.